contributors.md

```diff
@@ -65,6 +65,7 @@ The following people are not part of the development team, but have been contrib
 * Kelson Blakewood (spacek531) - title sequences for release versions 0.0.4 through 0.1.2, title sequences using milliseconds
 * Hugo Wallenburg (Goddesen) - Misc.
 * Matte Andersson (Nubbie) - Misc, UX
+* Helio Batimarqui (heliobatimarqui) - Misc.
```

Comment:
This list is kind of sorted by contribution date, so please move yours to the end

Reply:
Done.